### PR TITLE
Add test to validate PFN before and after softoffline

### PR DESCRIPTION
**Pull request by harish-24**
Patch adds test to validate PFN before and after softoffline

Signed-off-by: Harish <harish@linux.ibm.com>

### CODE REVIEW

**Comment by harish-24** [comment]
```
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-01-16T00.01-987e1a5/job.log
 (1/1) memory/numa_test.py:NumaTest.test_softoffline: PASS (4.24 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 4.94 s
```

**Comment by harish-24** [comment]
New commit:
```
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-01-16T23.25-d49de36/job.log
 (1/1) memory/numa_test.py:NumaTest.test_thp_compare: PASS (4.64 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 5.30 s
```

File: memory/numa_test.py [hunk 1]
```diff
@@ -98,6 +99,32 @@ def test(self):
...
+        Test PFN's before and after offlining
+        """
+        self.nr_pages = self.params.get(
+            'nr_pages', default=50)
+        os.chdir(self.teststmpdir)
+        self.log.info("Starting test...")
+        cmd = './softoffline -m %s -n %s' % (self.map_type, self.nr_pages)
+        ret = process.system(cmd, shell=True, sudo=True, ignore_status=True)
+        if ret != 0:
+            self.fail('Please check the logs for failure')
```
Comment (PraveenPenguin) [inline memory/numa_test.py]: having test case name listed in self.fail messge will be good idea

Comment (harish-24) [reply memory/numa_test.py]: Anyhow test will run in individual manner with respective failures, Is it enough or do we need to add testcase name?

Comment (PraveenPenguin) [reply memory/numa_test.py]: i think if we have test case name added should be easy to debug but i fine with this approach

**Pull request merged**